plugins/git/plugin.zsh

```diff
@@ -172,13 +176,17 @@ geometry_prompt_git_render() {
     fi
   fi
 
+  if git rev-parse --quiet --verify refs/stash >/dev/null; then
```

Comment:
I would check `PROMPT_GEOMETRY_GIT_SHOW_STASHES` here first

Reply:
Right. Ill update it